### ISSUE TITLE
The Gas Estimation says it will cost 0.01$ to mint a token, but in the end, it cost 30$+

### ISSUE THREAD

**Issue by jrainville** [status-im/status-desktop#13892]
# Bug Report

## Description

Experienced by Iuri when minting the tokens for the status community.

He was on mainnet Optimism. When minting a token, it said it would cost 0.0001 ETH or less than 0.01$, but when submitting, it actually failed with gas too low.

He ended up checking his past transactions and all of them, even though they said they would cost little, ended up costing ~30$
Here is an example: https://optimistic.etherscan.io/tx/0xa27026c0534649ada6d1ec7671fe746d88ed03aa4e7d90b9b892ecf68ff057be

If 0.01 ETH is the actual cost of the transaction, then we should present it to the user correctly in the app before minting, because right now, it comes as a shock.

If that cost is actually too high and it should have been 0.01$, then we need to fix that asap.

## Steps to reproduce

1. Be on mainnet Optimism
2. Mint a token (can be owner token)
3. See the estimated gas fee
4. Submit
5. See the fee at the end is way higher

#### Expected behavior

Either we fix the transaction so it costs what it shows, or we fix the estimation to show what it should really cost.

Bottom line is, the estimation should match the end fee (with a small discrepancy at best)

#### Actual behavior

The estimated fee and the paid fee are **wildly** different

### Additional Information

- Status desktop version: 2.27RC6

**Comment by alaibe** [comment]
Already work in progress
